validphys2/src/validphys/hyperoptplot.py

```diff
@@ -76,8 +91,12 @@ def params(self):
         apl = hyperparameters.get("activation_per_layer")
         if apl is not None and isinstance(apl, str) and "nodes_per_layer" in hyperparameters:
             # If apl is a string, it can only bring information if there is `nodes_per_layer`
-            apl = [apl]*(len(hyperparameters["nodes_per_layer"])-1) + ["linear"]
+            apl = [apl] * (len(hyperparameters["nodes_per_layer"]) - 1) + ["linear"]
             hyperparameters["activation_per_layer"] = apl
+        # 2. If there _was_ originally a reward included in this parameter (because it comes from and
+        #   previous hyperoptimization) mark it as such
+        #         if (reward:=hyperparameters.pop("reward", None)) is not None:
+        #             hyperparameters["old_reward"] = reward
```

Comment:
Might as well skip running this hyperparameter setup in that case, though I imagine this happens so rarely that it's really worth the effort to not do the simplest thing and just overwrite it. You might want to remove this comment though.